rita_common/src/network_endpoints/mod.rs

```diff
@@ -66,21 +65,24 @@ pub fn make_payments(pmt: (Json<PaymentTx>, HttpRequest)) -> HttpResponse {
     HttpResponse::Ok().json("Payment Received!")
 }
 
-pub fn hello_response(
-    req: (Json<LocalIdentity>, HttpRequest),
-) -> Result<Json<LocalIdentity>, RitaCommonError> {
+pub fn hello_response(req: (Json<LocalIdentity>, HttpRequest)) -> HttpResponse {
```

Comment:
keeping the result return type here is important. What about the error handling presented a problem with keeping it that way?